src/Host/Client/Impl/Extensions/RawEvalResultExtensions.cs

```diff
@@ -7,18 +7,9 @@
 
 namespace Microsoft.R.Host.Client.Extensions {
     public static class RawEvalResultExtensions {
-        public static bool HasRawData(this REvaluationResult result) {
-            return result.Raw?.Count > 0;
-        }
-
-        public static void SaveRawDataToFile(this REvaluationResult result, string filepath) {
-            if (result.HasRawData()) {
-                using (BinaryWriter writer = new BinaryWriter(File.OpenWrite(filepath))) {
-                    foreach (var data in result.Raw) {
-                        writer.Write(data);
-                    }
-                    writer.Flush();
-                }
+        public static void SaveRawDataToFile(this REvaluationResult result, string filePath) {
+            if (result.RawResult != null) {
+                File.WriteAllBytes(filePath, result.RawResult);
```

Comment:
Not here for sure, since it's just a thin wrapper (actually, I don't even know if this has much value now - before it dealt with multiple blobs, but now it's just `WriteAllBytes` under a different name). But call sites should be validated.